pages/wiki/creating-an-asteroid-app.md

```diff
@@ -152,10 +151,88 @@ Change the following `Environment` variables:
 
 Your app should now be able to run from the application when you click the start button in the bottom left sidebar.
 
+# Learn QML
+
+Main docs: https://doc.qt.io/ (AOS uses qt5)
+
+AOS specific component/API: https://github.com/AsteroidOS/qml-asteroid
+
+Examples of use of most of those can be found by searching the AsteroidOS codebase: https://github.com/search?q=org%3AAsteroidOS+StatusPage&type=code
+
+# Development Cycle
+
+Have 2 terminals:
+
+1. Do ssh and journalctl in one
+2. Other build app or vim
+
+QML Tester is an **on-watch** app to quickly test and debug qml's. You can install by running this command **on-watch**:
+
+```
+opkg install qmltester
+```
+
+Then you can Edit qmls by vim and scp
+
+```
+vim scp://user@myserver[:port]//path/to/file.txt
+```
+
+You can debug by reading the system logs by following along:
```

Comment:
This is a general tip that doesn't only apply to qmltester and should be under Tips and tricks